### ISSUE TITLE
MakeNSIS() does not support filepath for scripts with blanks in path name

### ISSUE THREAD

**Issue by heilingbrunner** [cake-build/cake#874]
### What You Are Seeing?

**Can't open script "C:\Path"**

### What is Expected?

MakeNSIS(...) should run with script file "C:\Path with blanks\setup.nsi"

### What version of Cake are you using?

0.11

### Are you running on a 32 or 64 bit system?

64 bit

### What environment are you running on?  Windows? Linux? Mac?

Windows 7

### Are you running on a CI Server?  If so, which one?

Travis CI

### How Did You Get This To Happen? (Steps to Reproduce)

Calling makensis.exe manually works with:
**"C:\Program Files (x86)\NSIS\makensis.exe" /DParam1=42 "C:\Path with blanks\setup.nsi"**
But using build.cake with
...
**MakeNSIS("C:\Path with blanks\setup.nsi", new MakeNSISSettings{Defines=defines, ToolPath=makensisPath});**
reports error:
**Can't open script "C:\Path"**

### Output Log

~~~sh
Error: Cake.Core.CakeException: MakeNSIS: Process returned an error (exit code 1).
   bei Cake.Core.Tooling.Tool`1.Run(TSettings settings, ProcessArgumentBuilder arguments, ProcessSettings processSettings, Action`1 postAction)
   bei Cake.Common.Tools.NSIS.MakeNSISRunner.Run(FilePath scriptFile, MakeNSISSettings settings)
   bei Submission#0.<.ctor>b__20()
   bei Cake.Core.ActionTask.Execute(ICakeContext context)
   bei Cake.Core.DefaultExecutionStrategy.Execute(CakeTask task, ICakeContext context)
   bei Cake.Core.CakeEngine.ExecuteTask(ICakeContext context, IExecutionStrategy strategy, Stopwatch stopWatch, CakeTask task, CakeReport report)
   bei Cake.Core.CakeEngine.RunTarget(ICakeContext context, IExecutionStrategy strategy, String target)
   bei Cake.Scripting.BuildScriptHost.RunTarget(String target)
   bei Submission#0.<.ctor>b__28()
   bei Cake.Core.ActionTask.Execute(ICakeContext context)
   bei Cake.Core.DefaultExecutionStrategy.Execute(CakeTask task, ICakeContext context)
   bei Cake.Core.CakeEngine.ExecuteTask(ICakeContext context, IExecutionStrategy strategy, Stopwatch stopWatch, CakeTask task, CakeReport report)
   bei Cake.Core.CakeEngine.RunTarget(ICakeContext context, IExecutionStrategy strategy, String target)
   bei Cake.Scripting.BuildScriptHost.RunTarget(String target)
   bei Submission#0..ctor(Session session, Object& submissionResult)
   bei Submission#0.<Factory>(Session session)
   bei Roslyn.Scripting.CommonScriptEngine.Execute[T](String code, String path, DiagnosticBag diagnostics, Session session, Boolean isInteractive)
   bei Roslyn.Scripting.Session.Execute(String code)
   bei Cake.Core.Scripting.ScriptRunner.Run(IScriptHost host, FilePath scriptPath, IDictionary`2 arguments)
   bei Cake.Commands.BuildCommand.Execute(CakeOptions options)
   bei Cake.CakeApplication.Run(CakeOptions options)
   bei Cake.Program.Main()

~~~

**Comment by gep13** [comment]
@heilingbrunner I am going to bump this to the next release as I am hoping to get the 0.12.0 release out tonight.  Thanks.

**Comment by devlead** [comment]
Fixed by #873
